src/RulesTest/RulesTests.csproj

```diff
@@ -1,203 +1,33 @@
-    <Compile Include="Library\HeadingLevelDescendsWhenNestedTest.cs" />
-    <Compile Include="Library\LocalizedLandmarkTypeNotCustomTests.cs" />
-    <Compile Include="Library\Structure\ControlView\ScrollbarTest.cs" />
-    <Compile Include="Library\Structure\ContentView\SpinnerTests.cs" />
-    <Compile Include="Library\Structure\ControlView\SpinnerTests.cs" />
-    <Compile Include="Library\UWPTest.cs" />
-    <Compile Include="PatternIDs.cs" />
-    <Compile Include="PropertyConditions\BoolPropertiesTest.cs" />
-    <Compile Include="PropertyConditions\IntPropertiesTests.cs" />
-    <Compile Include="PropertyConditions\ClickablePointTests.cs" />
-    <Compile Include="PropertyConditions\NameTest.cs" />
-    <Compile Include="PropertyConditions\BoundingRectangleTest.cs" />
-    <Compile Include="PropertyConditions\RelationshipsTest.cs" />
-    <Compile Include="PropertyConditions\SelectionItemPattern.cs" />
-    <Compile Include="PropertyConditions\SelectionPattern.cs" />
-    <Compile Include="PropertyConditions\StringPropertiesTest.cs" />
-    <Compile Include="PropertyConditions\ScrollPatternTest.cs" />
-    <Compile Include="Library\NameExcludesControlType.cs" />
-    <Compile Include="Library\NameExcludesSpecialCharacters.cs" />
-    <Compile Include="Library\NameIsReasonableLength.cs" />
-    <Compile Include="Library\BoundingRectangleContainedInParentTest.cs" />
-    <Compile Include="Library\BoundingRectangleDataFormatCorrectTest.cs" />
-    <Compile Include="Library\BoundingRectangleNotNullTest.cs" />
-    <Compile Include="Library\BoundingRectangleNotValidButOffScreenTest.cs" />
-    <Compile Include="Library\BoundingRectangleSizeReasonableTest.cs" />
-    <Compile Include="MockA11yElement.cs" />
-    <Compile Include="Library\NameIsNotEmpty.cs" />
-    <Compile Include="Library\NameIsNotNull.cs" />
-    <Compile Include="Library\NameIsNotWhiteSpace.cs" />
-    <Compile Include="Properties\AssemblyInfo.cs" />
-    <Compile Include="Library\NameIsInformative.cs" />
-    <Compile Include="PropertyConditions\PatternsTest.cs" />
-    <Compile Include="PropertyConditions\PlatformPropertiesTests.cs" />
-    <Compile Include="PropertyConditions\ContentViewTest.cs" />
-    <Compile Include="PropertyConditions\ControlViewTest.cs" />
-    <Compile Include="PropertyConditions\IntPropertyTest.cs" />
-    <Compile Include="MonsterTest.cs" />
-    <Compile Include="PropertyConditions\LandmarksTest.cs" />
-    <Compile Include="PropertyConditions\EnumPropertyUnitTests.cs" />
-    <Compile Include="PropertyConditions\ElementGroupsTests.cs" />
-    <Compile Include="PropertyConditions\ValueConditionUnitTests.cs" />
-    <Compile Include="RulesProviderTests.cs" />
+    <PackageReference Include="Microsoft.NET.Test.Sdk" Version="16.5.0-preview-20200102-01" />
+    <PackageReference Include="Moq" Version="4.13.1" />
+    <PackageReference Include="MSTest.TestAdapter" Version="2.1.0-beta2" />
```

Comment:
Would it make sense to use the non-prerelease assemblies? It looks like the prerelease versions got picked up in the project that is our template, but I'd prefer to use released versions, if possible. Post-change cleanup step, maybe?